test/Database/LSMTree/Model/Session.hs

```diff
@@ -233,9 +233,10 @@ runModelMWithInjectedErrors ::
   -> Model -> (Either Err a, Model)
 runModelMWithInjectedErrors Nothing onNoErrors _ st =
     runModelM onNoErrors st
-runModelMWithInjectedErrors (Just _) _ onErrors st =
+runModelMWithInjectedErrors (Just _errors) _ onErrors st =
     runModelM (onErrors >> throwError (ErrFsError "modelled FsError")) st
 
+
```

Comment:
```suggestion
```

Reply:
This feels like the kind of change we should leave up to formatters, rather than people.